jekyll/asset/script/main.js

```diff
@@ -189,15 +196,48 @@
 
             Object.keys(results.counts).forEach(function (type) {
                 if (counters[type]) {
-                    counters[type].number.innerHTML = results.counts[type];
-                    $(counters[type].circle).addClass("circle-" + type);
+                    counters[type].innerHTML = results.counts[type];
                 }
             });
+            $(document).trigger('update-totals');
         });
-    };
+    }
+
+    function totalOutputControl (element) {
+        var counters = {};
+        var countersArray = [];
+        $(element).find('[data-role=count]').each(function () {
```

Comment:
We should eventually store the different counts in memory instead of having to read them from the DOM whenever we want to aggregate. Not an issue for this pull request, but worth considering in the next refactor.